docs/docs/faq.md

```diff
@@ -82,9 +39,11 @@ conda config --set changeps1 true
 
 The `conda` config is located in the `/home/{user}/.condarc` file. You can change the conda config with a text editor (for example: `nano`, which is included in Nebari by default), and the changes will be applied on saving the file.
 
-## How do I clean up or delete the conda-store pod, if I need to?
+## How do I clean up old environment builds in conda-store?
+
+You may find that the pods hosting your environment get full over time, prompting you to clear them out. You can delete environments completely (including all builds of the environment) in the conda-store UI. Go to the environment, click `Edit` and then click `Delete`.
```

Comment:
Do you know what level of permissions are needed to do this?  I think you need to be an admin?

Reply:
Good point... I think "admin" in keyloak is required, but also with fine-grained permissions, you could get access through more specific permissions as well. I dont know the details on that though.

Reply:
I think I'll just mention `admin` access and leave the alternate permissions routes for the more detailed permissions documentation.